mkdocs.yml

```diff
@@ -0,0 +1,69 @@
+# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
+
+site_name: sigstore-python
+site_url: https://sigstore.github.io/sigstore-python
+repo_url: https://github.com/sigstore/sigstore-python
+repo_name: sigstore-python
+edit_uri: edit/main/docs/
+theme:
+  name: material
+  icon:
+    repo: fontawesome/brands/github
+  logo: assets/images/logo.png
+  features:
+    - content.action.edit
+    - header.autohide
+    - navigation.instant
+    - navigation.instant.progress
+    - navigation.footer
+  palette:
+    primary: custom
+  font:
+    text: Inter
+extra_css:
+  - stylesheets/custom.css
+nav:
+  - Home: index.md
+  - Installation: installation.md
+  - Signing: signing.md
+  - Verifying: verify.md
+  - Policy: policy.md
+  - Advanced:
+      - Custom Root of Trust: advanced/custom_trust.md
+      - Offline Verification: advanced/offline.md
+  # begin-api-section
+  - API:
```

Comment:
Nit: let's make this lowercase (`api`), since URLs are typically lowercase/insensitive.

Reply:
Updated 👍